apps/web/src/api/hooks/useInAppActivated.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { getInAppActivated } from '../integration';
+import { QueryKeys } from '../query.keys';
+
+interface IGetInAppActivatedResponse {
+  active: boolean;
+}
+
+export const useInAppActivated = () => {
```

Comment:
Could we have the return type of this hook?

Reply:
it's inferred from the `useQuery`